functions/get_puppetdb_hosts.pp

```diff
@@ -0,0 +1,17 @@
+function rsan::get_puppetdb_hosts() {
+  if $settings::storeconfigs {
+    $puppetdb_hosts =
+                puppetdb_query('resources[certname] {
+                    type = "Class" and
+                    title = "Puppet_enterprise::Profile::Puppetdb" and
+                    nodes {
+                      deactivated is null and
+                      expired is null
+                    }
+                  }').map |$data| { $data['certname'] }
+  } else {
+    $puppetdb_hosts = []
```

Comment:
puppetdb_hosts has to be returned in the correct format for 

 puppet_metrics_dashboard::puppetdb_list